db/services/cinema_hall.py

```diff
@@ -0,0 +1,14 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: str, hall_rows: int,
+                       hall_seats_in_row: int) -> CinemaHall:
+    return CinemaHall.objects.create(name=hall_name,
+                                     rows=hall_rows,
+                                     seats=hall_seats_in_row)
```

Comment:
There is a typo in the `create_cinema_hall` function. The field `seats` should be `seats_in_row` to match the `CinemaHall` model definition.